metadata/Cadastre/plss_quarter_quarter_sections.md

```diff
@@ -0,0 +1,79 @@
+# Title
+
+Utah PLSS Quarter Quarter Sections GCDB
+
+## ID
+
+4b63af59-da17-40dd-bf40-2996c489fc4d
+
+## Brief Summary
+
+Polygon dataset of the smallest unit of the Public Land Survey System (PLSS), where each unit represents 40 acres of land.
+
+## Summary
+
+This dataset represents the GIS Version of the Public Land Survey System. Geographic Coordinate Database ([GCDB](https://www.sciencebase.gov/catalog/item/4f4e4a7fe4b07f02db648dce)) base attribute data were used to create the second division feature class.
```

Comment:
**nitpick**: this summary feels awkward and doesn't seem to build or expand on the brief summary which is pretty good.

Reply:
I've reworded this, let me know what you think!